compiler/tests/input_files/program_input/main_multi_dimension_array.leo

```diff
@@ -0,0 +1,6 @@
+function main(x: [i16; (2, 2, 3)]){
+  console.log("x: {}", x);
+
+  let y: [i16; (2, 2, 3)] = [0i16; (2, 2, 3)];
+  console.log("y: {}", y);
```

Comment:
@gluax Could you have the Leo program include an assertion check `console.assert` so that we enforce that the array was indeed loaded correctly?